src/parquet/util/rle-encoding.h

```diff
@@ -268,6 +272,37 @@ inline bool RleDecoder::Get(T* val) {
 }
 
 template <typename T>
+inline uint32_t RleDecoder::GetBatch(T* values, uint32_t batch_size) {
```

Comment:
Nit: can you use `int` instead of `uint32_t` in this function (you may need to do `std::min<int>`) if it does not impact the performance?